main.go

```diff
@@ -46,6 +50,19 @@ func init() {
 func validateCompliance(_ *cobra.Command, args []string) {
 	setLogLevel(logLevel)
 
+	for module, licence := range overriddenModuleLicences {
+		cmd := exec.Command("go", "list", "-m", "-f", "\"{{.Dir}}\"", module)
+		cmd.Env = os.Environ()
+		cmd.Env = append(cmd.Env, "GO111MODULE=on")
```

Comment:
Is this necessary? Presumably this must be set anyway, by anyone calling `licence-compliance-checker --override-module-licence`